### DISCUSSION TITLE
头像显示问题

### DISCUSSION THREAD

**Discussion by eazhijia** [Tokinx/Wing#20] [讨论]
你好，能不能处理下头像不显示的问题呢，我尝试了下搜索到的一些方法，但是好像在我们这个主题上没用，也包括一些插件，比如WP First Letter Avatar（用用户名的第一个字母来显示头像）也是不能用，不知道是不是和主题不兼容的原因。

**Comment by Tokinx** [comment]
主题设置可以设置头像CDN的，比如使用 https://cravatar.cn/avatar 的头像加速服务
![image](https://user-images.githubusercontent.com/4526339/176369450-305970d2-082e-4b96-86e2-ada924a3a8e2.png)

**Comment by eazhijia** [comment]
好的，谢谢哈
